website/docs/d/cas_certificates.html.makrdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_cas_certificates"
+sidebar_current: "docs-alicloud-datasource-cas-certificates"
+description: |-
+    Provides a list of certs available to the user.
+---
```

Comment:
Just retain two blank in start of the line 6.

Reply:
done